### ISSUE TITLE
IntegrationTest Failure: io.crate.integrationtests.ArithmeticIntegrationTest

### ISSUE THREAD

**Issue by vibhutisawant** [crate/crate#7585]
**CrateDB version**:
working on master branch

**Environment description**:
JVM version: openjdk version "1.8.0_171"
Kernel: ppc64le GNU/Linux
Distribution: Ubuntu:16.04

**Problem description**:
Test Failure: io.crate.integrationtests.ArithmeticIntegrationTest fails for the following method:
testSelectWhereArithmeticScalarTwoReferenceArgs. The row count value returned by the query should be 2L,however it appears to be 1L.

**Steps to reproduce**:
./gradlew -Dtest.single=ArithmeticIntegrationTest sql:test --stacktrace

Please refer #7520 

I have a workaround to the test case failure:
In io.crate.integrationtests.ArithmeticIntegrationTest.java ,the following change is introduced at testSelectWhereArithmeticScalarTwoReferenceArgs function:
The output of the log function is rounded in order to compare it with the whole number only

The change was done due to the following reason
for the instance t(x,base)=t(9,3) the value of log(x,base) returned is 2.0000000000000004 instead of 2
PFA the images for reference 
![data present in table t](https://user-images.githubusercontent.com/41043754/43898719-57e37d82-9bfd-11e8-8832-8d5620b8c6d8.png)
![output of log guery](https://user-images.githubusercontent.com/41043754/43898720-5815d4b2-9bfd-11e8-864f-6f9a52810fb2.png)
![values](https://user-images.githubusercontent.com/41043754/43898721-5844e630-9bfd-11e8-9973-5ae1bf436c45.png)

Would it be advisable to introduce such a change?

**Comment by seut** [comment]
@vibhutisawant Yes that would be a feasible fix and will not change the behaviour of the test. Could you create a PR out of this? Also please sign our CLA if not done already. Thanks a lot for you contribution.

**Comment by vibhutisawant** [comment]
Thanks @seut .I'll get back to you on the PR.
